contrib/README.md

```diff
@@ -17,7 +17,23 @@ Make sure to also consider the following minor changes:
 * did you add any command flag?
 * did you fix a bug or improve any existing behavior?
 
-## Publishing a version
+## Release Process
+
+The CLI release process is automated now such that we will publish new builds when the `package.json` file is committed to `master` with a new `"version"`.
+
+### Managing the JIRA release
+
+Head over to the [JIRA release board](https://jira.mongodb.org/projects/REALMC?selectedItem=com.atlassian.jira.jira-projects-plugin%3Arelease-page&status=released-unreleased) and find the `cli-next` version.  Here you'll find all of the tickets ready to be released with this next version.
```

Comment:
we should add this to https://wiki.corp.mongodb.com/display/STITCH/Realm+Release+Manager+Workflow instead of here imo.

Reply:
happy to port it over there (actually I was thinking that same thing earlier). I ultimately landed here since the "release process" can be triggered at any point with a code change, so i figured it'd be helpful to have these details close by

Reply:
yeah, fair. I just figured for consistency is should at least have a link from that wiki page to here.